### DISCUSSION TITLE
LDAP test sync failure: Could not bind to LDAP

### DISCUSSION THREAD

**Discussion by R4junCajun** [snipe/snipe-it#12923] [Q&A]
Hello- Using the hosted plan and trying to enable LDAP sync to on-prem AD, but no matter what settings I set I am presented with "Could not bind to LDAP: Can't contact LDAP server" when trying the Test LDAP Synchronization. Any help or guidance is much appreciated. Relevant settings below for LDAP configuration.

![image](https://user-images.githubusercontent.com/131899406/234654061-f8958648-a247-454f-b06c-7e59db4ece78.png)
![image](https://user-images.githubusercontent.com/131899406/234654467-bc9921f0-3280-408a-a73c-d539058852e9.png)
![image](https://user-images.githubusercontent.com/131899406/234654541-b437a205-abff-48b2-a53a-5e3b5d0aaa8f.png)

Thanks!

**Comment by snipe** [comment]
Howdy! If it's saying it can't connect to the LDAP server, it usually means that there's a firewall in place that's not letting our servers talk to your LDAP server, and you may need to poke a hole in your firewall. Since you're hosted with us, you should open us an issue with us at support@snipeitapp.com and we can give you IP addresses, and help you work through getting set up.

**Reply by R4junCajun** [reply]
Super...thanks!
